CMakeLists.txt

```diff
@@ -448,7 +448,10 @@ set(CCF_NETWORK_TEST_ARGS
 )
 
 if(COMPILE_TARGET STREQUAL "sgx")
-  add_enclave_library(js_openenclave.enclave ${CCF_DIR}/src/js/openenclave.cpp)
+  add_enclave_library(
+    js_openenclave.enclave ${CCF_DIR}/src/js/openenclave.cpp
```

Comment:
Best to either split as you propose, or rename this to js_attestation_verification.

Reply:
I think splitting is better as verifying SGX quotes requires `open-enclave-hostverify` and we may want to build CCF with no dependency on OE whatsoever, but which can still verify SNP quotes.